docs/getting-started/data-views-in-sec.asciidoc

```diff
@@ -0,0 +1,37 @@
+[[data-views-in-sec]]
+= {data-sources-cap} in Elastic Security
+
+The active {data-source} determines what data appears throughout {es-sec} on pages with event or alert data.
+{Data-sources} are defined by the index patterns they include.
+Only data from the {es} {ref}/documents-indices.html[indices], {ref}/data-streams.html[data streams], or {ref}/alias.html[index aliases] specified in the active {data-source} will appear.
+
+IMPORTANT: Custom indices may not be included in the <<default-data-view-security, default {data source}>>. Modify it or create a custom {data-source} to include custom indices.
+
+[discrete]
+== Switch to another {data-source}
+
+You can tell which {data-source} is active by clicking the {data-source-cap} button that appears at the upper right of {es-sec} pages that display data, such as Overview, Alerts, or Hosts.
+To switch to another {data-source}, click `Choose {data-source}` and select one of the options.
+
+image::images/dataview-select-gif.gif[gif of opening the data view selection menu]
+
+[discrete]
+== Create or modify a {data-source}:
+
+To modify or delete an existing {data-source} or to define a new one, you need the required permissions.
+To learn more, refer to {apm-app-ref}/data-views.html[{kib} {data-sources-cap}].
+
+[discrete]
+[[default-data-view-security]]
+== The default {data-source}
+
+The first time you add data to an {es-sec} instance within a given {apm-app-ref}/xpack-spaces.html[space], the default {data-source} generates in that space and becomes active.
```

Comment:
I believe the initial Data View isn't generated until the first page visit within the Security Solution app -- so if data is ingested but no user ever goes to the Security Solution app there won't be a Data View just yet. @stephmilovic can you verify here please?